typings/vega-lite.d.ts

```diff
@@ -1,4 +1,3 @@
-declare module vl {
-  export var version: string;
-  export function compile(inputSpec: any, logger?: any): any;
-}
+export as namespace vl;
```

Comment:
Not just that. Notice how I am copying it to build as well. It will be useful for people who include the bundle instead of using a bunder.